src/main/java/org/reactome/release/goupdate/GoUpdateInstanceEditUtils.java

```diff
@@ -0,0 +1,103 @@
+package org.reactome.release.goupdate;
+
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.gk.model.GKInstance;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+/**
+ * Some utility methods for working with InstanceEdits for GO Update.
+ * @author sshorser
+ *
+ */
+class GoUpdateInstanceEditUtils
```

Comment:
Just curious - why are the class, inner enum and the enum's constructor package-private rather than public?

Reply:
There's no real need for these pieces of code to be exposed to the outside world, so why make it public?

Reply:
No reason - it just stuck out to me since everything I looked over was public.  It's fine for me to have them be package-private - I was just curious why :).